src/main/java/com/alienfast/bamboozled/ruby/config/RubyBuildConfigurationPlugin.java

```diff
@@ -85,6 +86,11 @@ public boolean isApplicableTo( Plan plan ) {
     }
 
     @Override
+    public boolean isApplicableTo(@NotNull ImmutablePlan plan) {
+        return false;
```

Comment:
@rosskevin, do you maybe know if we could also return `plan instanceof TopLevelPlan;` at the `ImmutablePlan plan`? I assume it should also work.